package.json

```diff
@@ -59,7 +50,8 @@
     "extends": "./nyc.config.js",
     "report-dir": "cypress-coverage",
     "reporter": [
```

Comment:
Can this key not come from `nyc.config.js`?

Also - amazing job! This is just about wrapped up.

Reply:
Thanks! 

I did try putting everything in ``nyc.config.js`` but ``report-dir`` and ``reporter`` would only work when specified in ``package.json``.